pyvista/plotting/plotting.py

```diff
@@ -1925,9 +1943,13 @@ def prepare_mapper(scalars):
         self.add_actor(actor, reset_camera=reset_camera, name=name, culling=culling,
                        pickable=pickable, render=render)
 
-        # Add scalar bar if available
-        if stitle is not None and show_scalar_bar and (not rgb or _custom_opac):
-            self.add_scalar_bar(title=stitle, **scalar_bar_args)
+        # hide scalar bar if using special scalars
+        if scalar_bar_args.get('title', None) == '__custom_rgba':
```

Comment:
Bikeshedding: `.get('title', None)` is slightly redundant, that's just `.get('title')`.

Reply:
Resolved in https://github.com/pyvista/pyvista/pull/1233/commits/48464cca32ca677e75a052b1dfa2e24983c26dc1